scripts/install

```diff
@@ -109,11 +111,23 @@ fi
 ynh_print_ON
 
 #=================================================
-# CONFIGURE JIRAFEAU
+# CREATE DATA DIRECTORY
 #=================================================
-ynh_script_progression --message="Configuring Jirafeau..." --weight=2
+ynh_script_progression --message="Creating a data directory..." --weight=1
 
 var_root=/home/yunohost.app/$app
+ynh_app_setting_set --app=$app --key=var_root --value=$var_root
```

Comment:
As `var_root` wasn't saved in the settings of the app, maybe it would be the good moment to switch the variable name to something more understandable like `datadir` or something else